tools/checkpoint_conversion/convert_bloom_checkpoints.py

```diff
@@ -12,51 +12,86 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import numpy as np
-import torch
-import transformers
-from absl import app
-from absl import flags
-from checkpoint_conversion_utils import get_md5_checksum
+import json
+import os
 
-from keras_nlp.models.bloom.bloom_backbone import BloomBackbone
+os.environ["KERAS_BACKEND"] = "torch"
+os.environ["CUDA_VISIBLE_DEVICES"] = "-1"
+
+import numpy as np  # noqa: E402
+import torch  # noqa: E402
+import transformers  # noqa: E402
+from absl import app  # noqa: E402
+from absl import flags  # noqa: E402
+from huggingface_hub import snapshot_download  # noqa: E402
```

Comment:
nit: just `import huggingface_hub` here (makes it easier to track what is coming from where)